ZelBack/src/services/appsService.js

```diff
@@ -4838,6 +4838,26 @@ async function verifyAppMessageSignature(type, version, appSpec, timestamp, sign
     if (timestamp > 1688947200000) {
       isValidSignature = signatureVerifier.verifySignature(messageToVerifyB, appSpec.owner, signature); // btc, eth
     }
+    // fix for repoauth / secrets order change for apps created after 1750273721000
+  } else if (isValidSignature !== true && appSpec.version === 7) {
```

Comment:
I think we should put version bellow 8, and check if the app specs have compose property. because we still have lots of apps on the network with older versions that have compose, so some might also need to be checked by this version.

Reply:
I see your other comment